components/bl-audio-recorder-component/src/index.js

```diff
@@ -16,6 +16,9 @@ export default function AudioRecorder({ component, eventHandlers, elRef }) {
 
   const [recordedBlob, setRecordedBlob] = useState();
   const [state, setState] = useState();
+  const [time, setTime] = useState(0);
+  const [timerInterval, setTimerInterval] = useState();
+  const [convertedTime, setConvertedTime] = useState();
```

Comment:
Also look's like we could leave only readable time. Or add an even't that will help us to stop recording at certain time. It could be implemented with even't that run's every recorded second.